R/plumber-step.R

```diff
@@ -195,6 +198,9 @@ PlumberEndpoint <- R6Class(
       if (!missing(serializer) && !is.null(serializer)){
         self$serializer <- serializer
       }
+      if (!missing(parsers) && !is.null(parsers)){
```

Comment:
```r
if (is.character(parsers)) {
  parsers <- get_parsers(parsers)
}
```
plumb-block creates its own list because it can use arguments

Reply:
￼ correct. I am converting plum block to produce a named list of alias to arguments. 

This will allow plum block and programmatic end points to create parsers with arguments￼￼. 

------

If you don't mind, I will tag you tomorrow morning to review the changes￼